### DISCUSSION TITLE
Windows Store can't install it.

### DISCUSSION THREAD

**Discussion by hy-dragon** [JanDeDobbeleer/oh-my-posh#3526] [General]
Windows Store can't install it.

**Comment by JanDeDobbeleer** [comment]
@dragon-hyl I'll need a little bit more context. What error are you seeing, what did you expect va what happens etc...

**Reply by hy-dragon** [reply]
> @dragon-hyl I'll need a little bit more context. What error are you seeing, what did you expect va what happens etc...
 I installed it form windows store.Now it is OK.But the windows terminal open slowly after I installed it.

**Reply by JanDeDobbeleer** [reply]
@dragon-hyl Windows Terminal or PowerShell? You need to make you don't have a virus scanner that blocks oh-my-posh's execution. We have a code signed binary, but not all virus scanners agree with that for some reason.

**Reply by hy-dragon** [reply]
Only a little delay,I  am already used to using it.
